README.md

```diff
@@ -519,6 +521,11 @@ These settings can be changed at any time.
   Bottlerocket enables the `json-file`, `awslogs`, and `none` drivers by default.
 * `settings.ecs.loglevel`: The level of verbosity for the ECS agent's logs.
   Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
+* `settings.ecs.metadata-service-steady-state-rate`: The steady state rate limit of the throttling configurations set for the task metadata service
+* `settings.ecs.metadata-service-burst-rate`: The burst rate limit of the throttling configurations set for the task metadata service
```

Comment:
I don't feel very strongly about this, but if we are going to invent new settings, I'd prefer to name them in a way that's consistent across Bottlerocket.

For kubelet we have similar fields:
```
settings.kubernetes.event-burst
settings.kubernetes.event-qps
settings.kubernetes.kube-api-burst
settings.kubernetes.kube-api-qps
```

Here the equivalent might be settings like:
```
settings.ecs.metadata-service-rps
settings.ecs.metadata-service-burst
```